content/spin/v3/serverless-ai-api-guide.md

```diff
@@ -32,19 +32,19 @@ ai_models = ["codellama-instruct"]
 // -- snip --
 ```
 
-> Spin supports "llama2-chat" and "codellama-instruct" for inferencing and "all-minilm-l6-v2" for generating embeddings.
+> Spin supports the models of the Llama architecture for inferencing and "all-minilm-l6-v2" for generating embeddings.
 
 ### File Structure
 
-By default, the Spin framework will expect any already trained model files (which are configured as per the previous section) to be downloaded by the user and made available inside a `.spin/ai-models/` file path of a given application. For example:
+By default, the Spin framework will expect any already trained model files (which are configured as per the previous section) to be downloaded by the user and made available inside a `.spin/ai-models/` file path of a given application. The files for each of the models need to be placed under a folder named after the architecture of the model (i.e) for `llama2-chat`, the folder needs to be named `llama`. For example:
```

Comment:
I got confused because the `llama` name doesn't match `llama2-chat` and this didn't seem to match the tutorial.  Here is my guess at what this is saying but I might be wrong.

```suggestion
By default, the Spin framework will expect any already trained model files (which are configured as per the previous section) to be downloaded by the user and made available inside a `.spin/ai-models/` file path of a given application.

Within the `.spin/ai-models` directory, models of the same architecture (e.g. `llama`) must be grouped under a directory with the same name as the architecture.

Within an architecture directory, each individual model (e.g. `llama2-chat`, `codellama-instruct`) must be placed under a folder with the same name as the model.  So for any given model, that files for the model are placed in the directory `.spin/ai-models/<architecture>/<model>`.  For example:
```

Although looking at `codellama-instruct` in the tutorial, maybe this isn't correct, because that seems to be a _single file_ directly in the `llama` directory.  So maybe we need to define the physical representation of a model (a file or directory) and then talk about how that is placed in the structure - would that be better?

Reply:
That suggested change is much clearer. Have made the changes